substrafl_examples/strategies_examples/plot_substrafl_fedavg.py

```diff
@@ -196,10 +119,44 @@ def MNISTraw2numpy(path: str, strict: bool = True) -> np.array:
     logs_permission=permissions,
 )
 
+dataset_keys = {}
+train_datasample_keys = {}
+test_datasample_keys = {}
+
+for ind, org_id in enumerate(ORGS_ID):
+    client = clients[org_id]
+
+    # Add the dataset to the client to provide access to the opener in each organization.
+    dataset_keys[org_id] = client.add_dataset(dataset)
+    assert dataset_keys[org_id], "Missing data manager key"
+
+    client = clients[org_id]
+
+    # Add the training data on each organization.
+    data_sample = DataSampleSpec(
+        data_manager_keys=[dataset_keys[org_id]],
+        test_only=False,
```

Comment:
`test_only` does not work anymore, let's remove it from the examples and we will remove it properly from substra soon after

Reply:
It is a required field from substra, we can't remove it before having deal with it in the sdk.